hooks/pre-commit

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 
 
```

Comment:
This shouldn't be here - will merge from dev once the PR is approved and merged that fixed git hooks